src/main/java/eu/lucaventuri/concurrent/AntiFreeze.java

```diff
@@ -99,46 +99,14 @@ public static void main(String[] args) throws InterruptedException {
 
         try {
             System.out.println("Starting...");
-            /*AntiFreeze frz = new AntiFreeze(1500, 7000);
```

Comment:
I might need this code if I need to test the anti freeze again, in different scenarios, so I am not sure I want to delete it.